Effisense/sensor-dashboard
# Create sensor view and database updates

Closes: #18 

## Checklist


## Changelog

- Create view for creating sensors, sensors/create.tsx
- Use react hook form to make input fields for creating a sensor 
-  On success, writes relevant data to database using a mutation in out tRPC backend (ctx.prisma.sensor.create(...))
- Added navbar 

## How to test

- First you need to delete the exisiting sensor in the prisma db, so run pnpm studio and delete the sensor
- Test by running the application and clicking on the add sensor button
- See that the form is appearing and write inside all the input form (latitute and longitude only allows numbers, I have not implemented an alert for this in the frontend yet but will do so when working on my next issue)
- For the ContainertypeId input you need to paste in the containerTypeId that is commented inside the create.tsx file under "defaultvalues"
- Then you can click add sensor, to check if it worked run pnpm studio inside the packages/db folder and see that a new sensor has appeared in the db
